### DISCUSSION TITLE
how to show countries in arabic when app locale is ar

### DISCUSSION THREAD

**Discussion by mennakhalilselim** [ysfkaya/filament-phone-input#52] [General]
![image](https://github.com/user-attachments/assets/6663ce8c-006b-4b26-9dbb-be4f7bc88425)

"ysfkaya/filament-phone-input": "^3.1"

**Comment by ysfkaya** [comment]
You can use it this way if there are specific countries you want to show;

https://github.com/ysfkaya/filament-phone-input?tab=readme-ov-file#only-countries

**Reply by mennakhalilselim** [reply]
I want a translation for all countries' names

**Reply by ysfkaya** [reply]
Try this way;

```php
PhoneInput::make('phone')->locale('ar')
```

https://github.com/ysfkaya/filament-phone-input?tab=readme-ov-file#locale

**Reply by mennakhalilselim** [reply]
this doesn't translate the countries
![image](https://github.com/user-attachments/assets/06a9307a-9cad-4e72-ba85-c490daf1ac1a)
![image](https://github.com/user-attachments/assets/d201833b-5d39-4da9-8f86-eaa7860434e2)

**Reply by ysfkaya** [reply]
It works for me.

![Screenshot 2024-10-01 at 14 09 02](https://github.com/user-attachments/assets/5c00b7bc-ae02-4cb6-8f04-c832b60cda9f)

Try

```bash
php artisan vendor:publish --tag="filament-phone-input-assets" --force
```

**Reply by mennakhalilselim** [reply]
I tried it and it didn't solve the problem, is any additional packages should I install?
I installed filament then
composer require ysfkaya/filament-phone-input
php artisan filament:assets
php artisan filament-phone-input:install

and I tried clearing the cache and reinstalled the package
